api/provider/twitter.go

```diff
@@ -77,6 +82,10 @@ func (t TwitterProvider) FetchUserData(ctx context.Context, tok *oauth.AccessTok
 	}
 	err = json.NewDecoder(bytes.NewReader(bits)).Decode(&u)
 
+	if u.Email == "" {
```

Comment:
yeah this is nice. Later we can try and detect Phone, and use that instead if it's there